README.md

```diff
@@ -24,104 +24,47 @@ or wiki—is a final product unless it is marked as such or appears on consumerf
-such as `cfgov-refresh` instead of `sheer` used in the Sheer installation instructions:
-
-```bash
-$ mkvirtualenv cfgov-refresh
-```
-
-Install the following dependencies into your virtual environment.
-We called ours `cfgov-refresh` (see previous step above):
-
-```bash
-$ workon cfgov-refresh
-
-$ pip install git+git://github.com/dpford/flask-govdelivery
-$ pip install git+git://github.com/rosskarchner/govdelivery
-```
-
-#### 2. Front-end setup
-
-The cfgov-refresh front-end currently uses the following frameworks / tools:
-
-- [Grunt](http://gruntjs.com): task management for pulling in assets,
-  linting and concatenating code, etc.
-- [Bower](http://bower.io): Package manager for front-end dependencies.
-- [Less](http://lesscss.org): CSS pre-processor.
-- [Capital Framework](https://cfpb.github.io/capital-framework/getting-started):
-  User interface pattern-library produced by the CFPB.
-
-**NOTE:** If you're new to Capital Framework, we encourage you to
-[start here](https://cfpb.github.io/capital-framework/getting-started).
-
-1. Install [Node.js](http://nodejs.org) however you'd like.
-2. Install [Grunt](http://gruntjs.com) and [Bower](http://bower.io):
-
-```bash
-$ npm install -g grunt-cli bower
-```
+Follow the instructions in [INSTALL](INSTALL.md).
 
 ## Configuration
 
-The project needs a number of environment variables.
-The project uses a WordPress API URL to pull in content
-and GovDelivery for running the subscription forms:
-
-- `WORDPRESS`(URL to WordPress install)
-- `GOVDELIVERY_BASE_URL`
-- `GOVDELIVERY_ACCOUNT_CODE` (GovDelivery account variable)
-- `GOVDELIVERY_USER` (GovDelivery account variable)
-- `GOVDELIVERY_PASSWORD` (GovDelivery account variable)
-- `SUBSCRIPTION_SUCCESS_URL` (Forwarding location on Subscription Success)
-
-> You can also export the above environment variables to your `.bash_profile`,
-or use your favorite alternative method of setting environment variables.
-
-**NOTE:** GovDelivery is a third-party web service that powers our subscription forms.
-Users may decide to swap this tool out for another third-party service.
-The application will function but throw an error if the above GovDelivery values are not set.
+For necessary server-side configurations, follow instructions in [Back-end-environment-variables](INSTALL.md#back-end-environment-variables).
 
 ## Usage
 
-Generally you will have four tabs (or windows) open in your terminal when using this project.
+Generally you will have four tabs (or windows)
+open in your terminal when using this project.
 These will be used for:
- 1. **Git operations**. Perform git operations and general development in the repository.
- 2. **Elasticsearch**. Run Elasticsearch instance.
- 3. **Sheer web server**. Run the web server.
- 4. **Grunt watch**. Run Grunt watch task for watching for changes to content.
+ 1. **Git operations**.
+    Perform git operations and general development in the repository.
+ 2. **Elasticsearch**.
+    Run an Elasticsearch instance.
```

Comment:
I'm not sure? This is a bit out of my realm of understanding.. My Elasticsearch appears to be running all the time using these directions, having it run on boot seems kind of overreaching for someone just checking out this project, but I don't really know what I'm taking about. Could it be a configuration footnote?